Presentation.Web/app/components/it-contract/tabs/it-contract-tab-data-processing.controller.ts

```diff
@@ -0,0 +1,97 @@
+﻿module Kitos.DataProcessing.Registration.Edit.Ref {
+    "use strict";
+
+    export class EditDataProcessingContractController {
+        static $inject: Array<string> = [
+            "hasWriteAccess",
+            "contract",
+            "ItContractsService",
+            "bindingService",
+            "apiUseCaseFactory"
+        ];
+
+        private readonly contractId: number;
+        constructor(
+            public hasWriteAccess,
+            private readonly contract: Kitos.Models.ItContract.IItContractDTO,
+            private readonly contractService: Kitos.Services.Contract.IItContractsService,
+            private readonly bindingService: Kitos.Services.Generic.IBindingService,
+            private readonly apiUseCaseFactory: Services.Generic.IApiUseCaseFactory
+        ) {
+            this.contractId = this.contract.id;
+
+            this.bindDataProcessingRegistrations();
+        }
+
+        dataProcessingRegistrations: Models.ViewModel.Generic.IMultipleSelectionWithSelect2ConfigViewModel<Models.Generic.NamedEntity.NamedEntityDTO>;
+
+        private bindDataProcessingRegistrations() {
+            this.bindingService.bindMultiSelectConfiguration<Models.Generic.NamedEntity.NamedEntityDTO>(
+                config => this.dataProcessingRegistrations = config,
+                () => this.contract.dataProcessingRegistrations,
+                element => this.removeDataProcessingRegistration(element.id),
+                newElement => this.addDataProcessingRegistration(newElement),
+                this.hasWriteAccess,
+                this.hasWriteAccess,
```

Comment:
2x this.hasWriteAccess

Reply:
Ja den ene er for at fortælle om man må tilføje og den anden for om man må slette. Lavet på samme måde under DataProcessingRegistration main